repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/contract/ContractPreProcessor.java

```diff
@@ -193,6 +193,16 @@ private void updateExistingVersion(EntityMutationContext context, AtlasEntity en
 
         context.addUpdated(entity.getGuid(), entity, entityType, vertex);
 
+        AtlasEntityComparator entityComparator = new AtlasEntityComparator(typeRegistry, entityRetriever, context.getGuidAssignments(), true, true);
```

Comment:
Can you please check & confirm whether we are getting the same info in the audit logs as well, you can run auditsearch API to fetch entity audits, you must have a details attribute which logs the same info that Kafka notfification’s mutatedDetails